modules/README.md

```diff
@@ -192,6 +192,10 @@ Modules that are still in development and less stable are labeled with the
 * **[spack-install]** ![community-badge] ![experimental-badge] : Creates a
   startup script to install [Spack](https://github.com/spack/spack) on an
   instance or a slurm login or controller.
+* **[ramble-setup]** ![community-badge] ![experimental-badge] : Creates a
```

Comment:
You need to add an reference with the link to the module. See L202 for examples.